### PR TITLE
Ava GUI: Make Dialogue More Intuitive

### PR DESCRIPTION
**Pull request by IsaacMarovitz**
Adjusts several aspects of the settings dialogue to make it more intuitive and standard.

- `Save` renamed to `OK`
- `Close` renamed to `Cancel`
- Layout of buttons adjusted to establish a clearer visual hierarchy (Adapts to match OS)
- Made `OK` default action (shortcut with `Enter`) and accented it
- Added `Escape` shortcut to `Cancel` button

Screenshot 2022-11-30 at 11 33 10

### CODE REVIEW

**Review by EmulationFanatic — APPROVED** [review]
Verbiage and appearance LGTM. I'll leave it to another to review the method :)
Thanks!

File: Ryujinx.Ava/Ui/ViewModels/SettingsViewModel.cs [hunk 1]
```diff
@@ -474,11 +479,40 @@ public void SaveSettings()
             MainWindow.UpdateGraphicsConfig();
 
             _previousVolumeLevel = Volume;
+
+            if (_owner is SettingsWindow owner)
```
Comment (AcK77) [inline Ryujinx.Ava/Ui/ViewModels/SettingsViewModel.cs:483]: Isn’t `_owner` type always `SettingsWindow` since it’s the model class?

Comment (IsaacMarovitz) [reply Ryujinx.Ava/Ui/ViewModels/SettingsViewModel.cs:483]: `_owner` is type `StyleableWindow`. I can just do `_owner as SettingsWindow`, but that is a possible `NullReferenceException`/

**Review by AcK77 — APPROVED** [review]

**Review by marysaka — APPROVED** [review]

**Comment by Squall-Leonhart** [comment]
Introduced a regression on the Controller Applet

"SettingsButtonClose"
image

**Pull request merged**